static/app/components/seenByList.stories.tsx

```diff
@@ -0,0 +1,151 @@
+import {Fragment, useEffect} from 'react';
+
+import Placeholder from 'sentry/components/placeholder';
+import SeenByList from 'sentry/components/seenByList';
+import JSXProperty from 'sentry/components/stories/jsxProperty';
+import Matrix from 'sentry/components/stories/matrix';
+import SizingWindow from 'sentry/components/stories/sizingWindow';
+import storyBook from 'sentry/stories/storyBook';
+import {useMembers} from 'sentry/utils/useMembers';
+import {useUser} from 'sentry/utils/useUser';
+
+function useLoadedMembers() {
+  const {members, loadMore, ...rest} = useMembers();
+
+  useEffect(() => {
+    // `loadMore` is not referentially stable, so we cannot include it in the dependencies array
+    loadMore();
+  }, []); // eslint-disable-line react-hooks/exhaustive-deps
+
+  return {members, loadMore, ...rest};
+}
+
+export default storyBook(SeenByList, story => {
+  story('Default', () => {
+    const {members, fetching} = useLoadedMembers();
+
+    return (
+      <SizingWindow display="block" style={{width: '50%'}}>
+        {fetching ? <Placeholder /> : <SeenByList seenBy={members} />}
+      </SizingWindow>
+    );
+  });
+
+  story('iconTooltip', () => {
+    const {members, fetching} = useLoadedMembers();
+
+    return (
+      <Fragment>
+        <p>
+          Default is{' '}
+          <JSXProperty name="iconTooltip" value="People who have viewed this" />
+        </p>
+        <SizingWindow display="block" style={{width: '50%'}}>
+          {fetching ? (
+            <Placeholder />
+          ) : (
+            <SeenByList
+              seenBy={members}
+              iconTooltip="These folks have all seen this record"
+            />
+          )}
+        </SizingWindow>
+      </Fragment>
+    );
+  });
+
+  story('Shows users except yourself', () => {
```

Comment:
```suggestion
  story('Always shows users except yourself', () => {
```
to clarify that the behavior is always this